src/components/devtools/TransferIO.tsx

```diff
@@ -42,7 +47,10 @@ function TransferIO() {
   async function confirmTransfer() {
     try {
       setTransfering(true);
-      if (isArweaveTransactionID(toAddress.trim())) {
+      if (
+        isEthAddress(toAddress.trim()) ||
```

Comment:
May a single util would be good here. We may add solana and other chains in the future.